docs/source/algorithms/LoadNexus-v1.rst

```diff
@@ -12,9 +12,25 @@ Description
 The algorithm LoadNexus will read the given Nexus file and try to
 identify its type so that it can be read into a workspace. The file name
 can be an absolute or relative path and should have the extension .nxs
-or .nx5. Currently only Nexus Muon Version 1 files are recognised, but
-this will be extended as other types are supported such as
-:ref:`algm-LoadNexusProcessed`.
+or .nx5.
+
+The type of Nexus file is identified as follows:
+
+* If the file has a group of class ``SDS`` of name ``"definition"`` or ``"analysis"`` 
```

Comment:
Is "analysis" really checked for? I see these confusing comments in `LoadNexus::exec()`:
```
  // Test the given filename to see if it contains the field "analysis" with                                                                                                                                                                 
  // value "muonTD"                                                                                                                                                                                                                          
  // within the first NXentry.                                                                                                                                                                                                               
  // If so, assume it is a Muon Nexus file (version 1) and pass to the                                                                                                                                                                       
  // LoadMuonNexus algorithm                                                                                                                                                                                                                 
  // Otherwise try LoadIsisNexus.                                                                                                                                                                                                            

  // FIXME: The following variable isn't used, but the above comments                                                                                                                                                                        
  // imply it should be.                                                                                                                                                                                                                     
  // std::string dataName="analysis";                                                                                                                                                  
```

And the code seems to check the `definition` only.